CONTRIBUTOR-DOCS/04_workstream-info/02_2nd-gen-component-migration/02_step-by-step/05_migrate-rendering-and-styles.md

```diff
@@ -0,0 +1,139 @@
+<!-- Generated breadcrumbs - DO NOT EDIT -->
+
+[CONTRIBUTOR-DOCS](../../../README.md) / [Workstream Info](../../README.md) / [2nd-gen Component Migration](../README.md) / Step By Step / Migrate rendering & styles from Spectrum CSS
+
+<!-- Document title (editable) -->
+
+# Migrate rendering & styles from Spectrum CSS
+
+<!-- Generated TOC - DO NOT EDIT -->
+
+<details open>
+<summary><strong>In this doc</strong></summary>
+
+- [[TODO: Integrate this content]](#-todo-integrate-this-content-)
+- [Bring over styles from Spectrum CSS](#bring-over-styles-from-spectrum-css)
+- [Update styles in the 2nd-generation component](#update-styles-in-the-2nd-generation-component)
+
+</details>
+
+<!-- Document content (editable) -->
+
+## [ TODO: Integrate this content ]
+
+The following bits from the top-level migration guide outline need to be integrated into this doc:
+
+> - In RENDERING & STYLING section:
+>     - Add static `styles` getter returning the S2 stylesheet
+>     - Implement the `render()` method with S2 rendering logic
+> - Create the S2 stylesheet (`[component].css`)
+
+## Bring over styles from Spectrum CSS
+
+Now we can bring over styles from the `spectrum-two` package in Spectrum CSS. Start by checking out the `@adobe/spectrum-css` repository and pulling down the `spectrum-two` branch.
+
+Identify the component you need to migrate by searching the `components` directory for the component name.
+
+Inside the CSS directory, you can expect to find the following files:
+
+- `index.css` - This is the source of truth for the component Styles
+- `dist/index.css` - This is the processed version of the component styles; for now, this is the best place to source styles for SWC. When copying files from Spectrum CSS into the 2nd-generation component, using this file will ensure you get the benefits of the CSS build tooling.
```

Comment:
This might be obvious, but we don't commit `dist/index.css` so you have to install & build locally to access this file.

Reply:
great callout, we should add that context to the intro line